server/src-lib/Hasura/GraphQL/Transport/WebSocket.hs

```diff
@@ -343,25 +346,40 @@ onStart serverEnv wsConn (StartMsg opId q) = catchAndIgnore $ do
       E.GQFieldResolvedHasura (E.ExOpSubs lqOp) -> Just lqOp
       _ -> Nothing
 
-    execQueryOrMut reqId query genSql action = do
-      -- log the generated SQL and the graphql query
-      -- liftIO $ logGraphqlQuery logger $ QueryLog query genSql reqId
+    execQueryOrMut genSql action = do
+      addToLog
       resp <- liftIO $ runExceptT action
       liftEither resp
-
-    runRemoteGQ :: (MonadError QErr m, MonadIO m)
-      => E.ExecutionCtx -> RequestId -> UserInfo -> [H.Header]
+      where
+        addToLog =
```

Comment:
I guess this should be exposed at the top level (you probably had plans to do that already)